src/services/session-cache.ts

```diff
@@ -48,6 +49,28 @@ export default class SessionCache {
       .filter(p => options?.created === undefined || !!p.isCreated === options.created);
   };
 
+  public getPlacesForDisplay = (options?: {
+    type?: string;
```

Comment:
can you make this a reusable type shared by `getPlacesForDisplay` and `getPlaces`?